src/instructlab/model/train.py

```diff
@@ -201,8 +205,7 @@ def train(
     **kwargs,  # pylint: disable=unused-argument
 ):
     """
-    Takes synthetic data generated locally with `ilab data generate` and the previous model and learns a new model using the MLX API.
-    On success, writes newly learned model to {model_dir}/mlx_model, which is where `chatmlx` will look for a model.
+    Takes synthetic data generated locally with `ilab data generate` and the previous model and learns a new model.
```

Comment:
Why did you change this? Was it incorrect?

Reply:
Training support linux and macos(maybe other platforms too).  On linux, it does not use MLX. 
These two sentences both indicate training is for macOS only. The output path for macOS is not right, I don't think it's necessary to mention output directory here. When user start training, it will output the message.


Some description from readme:
`ilab model train` outputs a brand-new model that is saved in the `<model_name>-mlx-q` directory called `adapters.npz` (in `Numpy` compressed array format). For example:

```shell
(venv) $ ls instructlab-merlinite-7b-lab-mlx-q
adapters-010.npz        adapters-050.npz        adapters-090.npz        config.json             tokenizer.model
adapters-020.npz        adapters-060.npz        adapters-100.npz        model.safetensors       tokenizer_config.json
adapters-030.npz        adapters-070.npz        adapters.npz            special_tokens_map.json
adapters-040.npz        adapters-080.npz        added_tokens.json       tokenizer.jso
```